util/src/main/scala/org/typelevel/catbird/util/asyncstream.scala

```diff
@@ -21,7 +21,7 @@ trait AsyncStreamInstances extends AsyncStreamInstances1 {
 
   final def asyncStreamEq[A](atMost: Duration)(implicit A: Eq[A]): Eq[AsyncStream[A]] = new Eq[AsyncStream[A]] {
     final def eqv(x: AsyncStream[A], y: AsyncStream[A]): scala.Boolean = Await.result(
-      x.take(1).toSeq().join(y.take(1).toSeq()).map { case (x, y) => x == y },
+      x.head.join(y.head).map((Eq[Option[A]].eqv _).tupled),
```

Comment:
I made this change because with the additional compiler warnings, the compiler was complaining that the `implicit A: Eq[A]` was unused. This is a 2.12/2.13 cross-compatible way to check equality using the `Eq[A]` instance.